language/move-prover/spec-lang/src/translate.rs

```diff
@@ -1160,8 +1160,12 @@ impl<'env, 'translator> ModuleTranslator<'env, 'translator> {
             used_memory: BTreeSet::new(),
             uninterpreted: false,
             is_move_fun: true,
+            is_native: false,
             body: None,
         };
+        if let EA::FunctionBody_::Native = def.body.value {
```

Comment:
nit: `fun_decl.is_native = matches!(def.body.value, EA::FunctionBody_::Native)` is nicer. Or you use this right away in the constructor above and then do not need the `mut`.